scripts/snapshot-restore-balance/snapshot-balances.js

```diff
@@ -72,28 +103,81 @@ async function restoreBalances(api, keyring, snapshotBalancesFile, custodySeed)
     }
 }
 
+function usage() {
+    console.error(`Usage : snapshot or restore
+    node snapshot-balances.js [-e ] snapshot
```

Comment:
I'm afraid that having only the `-e` option might be a bit confusing. I'd suggest to either have all the options here, or none.

Reply:
It's a leftover... thanks!